src/python/WMCore/Database/DBCore.py

```diff
@@ -134,6 +147,8 @@ def processData(self, sqlstmt, binds={}, conn=None,
                 connection = self.connection()
             else:
                 connection = conn
+            if  isinstance(self.engine.dialect, OracleDialect_cx_oracle):
```

Comment:
Valentin, I know you like leaving two white spaces in an `if` clause, unfortunately it generates one more complaint line in the pylint output. You just told me you'll probably not have time today, thus please avoid them in future PRs.